### DISCUSSION TITLE
Any ideas, solutions for this error?  "Failed to toggle replication - must be owner of publication supabase_realtime"

### DISCUSSION THREAD

**Discussion by aleskozelsky** [supabase/supabase#13321] [Questions]
Hello, 

I am following [NextJS Subscription Starter](https://github.com/vercel/nextjs-subscription-payments) and all was working fine. 

Today I wanted to play around with creating a custom table and using Realtime to display some live data on the front end. 

When I do any changes to any settings in the **Database** -> **Replication**  I am getting the following error: 

Failed to toggle replication for my_custom_table_name: failed to update pg.publications with the given ID: must be owner of publication supabase_realtime

Any ideas what this may be cased by? 

Thank you in advance, 

Ales

**Comment by w3b6x9** [comment]
@aleskozelsky you're using a hosted Supabase project, correct? can you create a support ticket so we can take a look at your project. it's most likely project-specific.

**Reply by aleskozelsky** [reply]
hi @w3b6x9, 

Thanks for your quick response. 

Yes I am using hosted solution (supabase.com) and I have already raised a ticket (1525995839) for this. Within that ticket there was a link in this forum, so I posted here too, as I am a free user. 

And Yes, I believe it is project specific. I am just starting out, basically I have run the [.sql query from the NextJS Supabase starter](https://github.com/vercel/nextjs-subscription-payments/blob/main/schema.sql) added a few dummy users, played around a little bit, and then when I came back to it a few days later, I was unable to change any Realtime settings as per post above. 

As part of the investigation, I have created a new project (exact same steps) and run the same .sql query all was working fine on the other project 

Thanks for any help and suggestions,

have a nice day,

Ales
